2021-11-24 22:18:51,808 INFO    MainThread:43124 [wandb_setup.py:_flush():71] setting env: {}
2021-11-24 22:18:51,808 INFO    MainThread:43124 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-24 22:18:51,808 INFO    MainThread:43124 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2021-11-24 22:18:51,808 INFO    MainThread:43124 [wandb_init.py:_log_setup():371] Logging user logs to /home/snamburi/Debarshi/wandb/run-20211124_221851-1z0m9ffj/logs/debug.log
2021-11-24 22:18:51,808 INFO    MainThread:43124 [wandb_init.py:_log_setup():372] Logging internal logs to /home/snamburi/Debarshi/wandb/run-20211124_221851-1z0m9ffj/logs/debug-internal.log
2021-11-24 22:18:51,809 INFO    MainThread:43124 [wandb_init.py:init():404] calling init triggers
2021-11-24 22:18:51,809 INFO    MainThread:43124 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-24 22:18:51,809 INFO    MainThread:43124 [wandb_init.py:init():449] starting backend
2021-11-24 22:18:51,809 INFO    MainThread:43124 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-24 22:18:51,847 INFO    MainThread:43124 [backend.py:ensure_launched():199] starting backend process...
2021-11-24 22:18:51,868 INFO    MainThread:43124 [backend.py:ensure_launched():204] started backend process with pid: 43134
2021-11-24 22:18:51,869 INFO    MainThread:43124 [wandb_init.py:init():458] backend started and connected
2021-11-24 22:18:51,871 INFO    MainThread:43124 [wandb_init.py:init():519] updated telemetry
2021-11-24 22:18:51,872 INFO    MainThread:43124 [wandb_init.py:init():549] communicating current version
2021-11-24 22:18:52,676 INFO    MainThread:43124 [wandb_init.py:init():554] got version response 
2021-11-24 22:18:52,677 INFO    MainThread:43124 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-24 22:18:52,887 INFO    MainThread:43124 [wandb_init.py:init():592] starting run threads in backend
2021-11-24 22:18:54,705 INFO    MainThread:43124 [wandb_run.py:_console_start():1816] atexit reg
2021-11-24 22:18:54,705 INFO    MainThread:43124 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-24 22:18:54,706 INFO    MainThread:43124 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-24 22:18:54,710 INFO    MainThread:43124 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-24 22:18:54,710 INFO    MainThread:43124 [wandb_init.py:init():619] run started, returning control to user process
2021-11-24 22:18:54,711 INFO    MainThread:43124 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-24 22:18:54,832 INFO    MainThread:43124 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-24 22:18:54,833 INFO    MainThread:43124 [wandb_run.py:_restore():1758] restore
2021-11-24 22:18:57,240 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:18:57,344 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:18:57,459 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:18:57,931 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 11054
}

2021-11-24 22:18:58,042 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 11054
}

2021-11-24 22:18:58,145 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}

2021-11-24 22:18:58,248 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}

2021-11-24 22:18:58,351 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}

2021-11-24 22:18:58,453 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}

2021-11-24 22:18:58,652 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}

2021-11-24 22:18:58,917 INFO    MainThread:43124 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11054
  total_bytes: 11054
}
local_info {
}

2021-11-24 22:19:00,101 INFO    MainThread:43124 [wandb_run.py:_append_files():2184] logging synced files
